orbit_determination/ekf.py

```diff
@@ -84,6 +90,8 @@ def __init__(
 
         self.cond_threshold = 1e15
         self.H = np.append(np.zeros((1, 3)), np.eye(3), axis=0)
+        self.config = config
+        self.data_manager = data_manager
 
     def predict(self, u: np.ndarray) -> None:
```

Comment:
The rest of the ekf member functions other than this one take the `data_manager` as a parameter instead of using the member variable. I think either approach is fine, but we should be consistent throughout.